### PR TITLE
INTGR-165: Developing Add-Ins Page

### PR DESCRIPTION
**Pull request by tranjennie**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-import {Header, Navbar} from "./components";
+import { Header, Navbar } from "./components";
 import LandingPage from "./pages";
 
 import MyGeotabIntroduction from "./pages/myGeotab/introduction";
 import MyGeotabWhatsNew from "./pages/myGeotab/whatsNew";
 import MyGeotabGettingStarted from "./pages/myGeotab/guides/gettingStarted";
 import usingCustomTelematicsDevices from "./pages/myGeotab/guides/usingCustomTelematicsDevices";
 import UsingGoDevices from "./pages/myGeotab/guides/usingGoDevices";
+import MyGeotabDevelopingAddIns from "./pages/myGeotab/guides/developingAddIns";
```
Comment (kmets89) [inline src/App.tsx:10]: A link also needs to be added to the Navbar for this page

File: src/pages/myGeotab/guides/developingAddIns.tsx [hunk 2]
```diff
@@ -0,0 +1,848 @@
...
+
+
+export default function DevelopingAddIns() {
+    return (
+        <div className="grayBackground pageContent">
+            <h1 className="title">Developing Add-Ins</h1>
+            <div className="paragraph">
+                <p>Add-Ins are used to extend the functionality provided by MyGeotab and Geotab Drive. An Add-In is JavaScript, HTML and CSS loaded into the MyGeotab or Geotab Drive portal and resides directly inside the user interface. This allows third-parties to create a seamless user experience and provide solutions that would otherwise require the user to visit a different website altogether. <a href="https://github.com/Geotab/sdk-addin-samples">Click here</a> to find the sample Add-Ins.</p>
+                <div className="side-note">
+                    <p>The add-in generator is a great developer tool that allows integrators to create scaffolded add-in projects. You can learn more about the generator and all of its features by going into the <a href="https://github.com/Geotab/generator-addin">generator-addin repository</a>.</p>
```
Comment (kmets89) [inline src/pages/myGeotab/guides/developingAddIns.tsx:831]: for the sections with the left-hand-side blue bar, as discussed in standup today we'll put those in the blue informational boxes as soon as that component is ready from the myG team. Can you add TODOs for now where all of those go so they're easy to fix later?

**Review by kmets89 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (tranjennie) [reply src/App.tsx:10]: Updated navbar

File: src/pages/myGeotab/guides/developingAddIns.tsx (hunk 2 above)
Comment (tranjennie) [reply src/pages/myGeotab/guides/developingAddIns.tsx:831]: Created a component for informational box

**Pull request merged**